modules/interop-specs2/test/src/weaver/interop/specs2/MatchersSpec.scala

```diff
@@ -0,0 +1,25 @@
+package weaver.interop.specs2
+
+import weaver.SimpleIOSuite
+
+object MatchersSpec extends SimpleIOSuite with IOMatchers {
```

Comment:
There's a [DogFood](https://github.com/disneystreaming/weaver-test/blob/master/modules/framework/src/weaver/framework/DogFood.scala) trait that contains the necessary functions to manually run the framework (it's very meta) and intercept the events it fires as well as the logs, which could be used to test the negative cases, but it involves a fair amount of string matching. 

[Example usage](https://github.com/disneystreaming/weaver-test/blob/master/modules/framework/test/src/DogFoodTests.scala#L78) 

Alternatively, you could tap into the [spec](https://github.com/disneystreaming/weaver-test/blob/master/modules/core/src/weaver/suites.scala#L24) method (nesting a suite containing failing tests inside another object to prevent the actual framework instance from detecting it). 

That'd let you test that conversions are actually capturing the correct sourcecode metadata.